src/api/entities/Account/MultiSig/index.ts

```diff
@@ -113,6 +123,44 @@ export class MultiSig extends Account {
     );
   }
 
+  /**
+   * Return all { @link api/entities/HistoricalMultiSigProposal!HistoricalMultiSigProposal } for this MultiSig Account
+   */
+  public async getHistoricalProposals(opts: {
+    size?: BigNumber;
+    start?: BigNumber;
+  }): Promise<ResultSet<HistoricalMultiSigProposal>> {
```

Comment:
IMO, we still don't need a `HistoricalMultiSigProposal` entity. They way you are doing is just creating a replica of `MultiSigProposal` as `HistoricalMultiSigProposal` and adding wrapper methods to call `MultiSigProposal`. Instead you can change the return type directly here to `Promise<ResultSet<MultiSigProposal>>` or like we do with other entities, call the SQ and return a detailed information in a interface call `HistoricalMultiSigProposal` which will have all the details of multisig fetched beforehand